### PR TITLE
NIFI-3701 Documentation improvements for 1.x

### PR DESCRIPTION
**Pull request by andrewmlim**
Admin Guide:

- Corrected/updated access policy tables to better align with UI
- Split policy tables into one for Global, one for Component
- Corrected empty authorizations.xml and users.xml cluster usage
- Updated description for nifi.templates.directory properties description
- Added note about using List/Delete Queues when clustered

User Guide:

- Added note about Back Pressure defaults
- Updated "Configure Site-to-Site server NiFi instance" section as it referenced 0.x behavior

### CODE REVIEW

**Comment by alopresto** [comment]
Reviewing...

File: nifi-docs/src/main/asciidoc/user-guide.adoc [hunk 1]
```diff
@@ -834,6 +834,7 @@ image:connection-settings.png["Connection Settings"]
 The Connection name is optional. If not specified, the name shown for the Connection will be names of the Relationships
 that are active for the Connection.
 
+===== FlowFile Expiration
 File expiration is a concept by which data that cannot be processed in a timely fashion can be automatically removed from the flow.
```
Comment (alopresto) [inline nifi-docs/src/main/asciidoc/user-guide.adoc]: I think the first word here should be *FlowFile* -- *FlowFile expiration is a concept...*.

Comment (andrewmlim) [inline nifi-docs/src/main/asciidoc/user-guide.adoc]: Agreed.

File: nifi-docs/src/main/asciidoc/administration-guide.adoc [hunk 2]
```diff
@@ -462,30 +462,39 @@ Here is an example entry:
 
 After you have edited and saved the 'authorizers.xml' file, restart NiFi. Users and roles from the 'authorized-users.xml' file are converted and added as identities and policies in the 'users.xml' and 'authorizations.xml' files.  Once the application starts, users who previously had a legacy Administrator role can access the UI and begin managing users, groups, and policies.
 
-Here is a summary of policies assigned to each legacy role if the NiFi instance has an existing flow.xml.gz:
+The following tables summarize the global and component policies assigned to each legacy role if the NiFi instance has an existing flow.xml.gz:
```
Comment (alopresto) [inline nifi-docs/src/main/asciidoc/administration-guide.adoc]: It appears the standard is to note filenames via a single quote, which makes it italic. Suggest changing `flow.xml.gz` to `'flow.xml.gz'`.

Comment (andrewmlim) [inline nifi-docs/src/main/asciidoc/administration-guide.adoc]: Good catch!  Thanks!

**Comment by alopresto** [comment]
I will make the two minor fixes above and merge. Thanks @andrewmlim .

**Comment by alopresto** [comment]
I ran `contrib-check` and verified that the generated docs look good. Merged to master.

**Comment by alopresto** [comment]
Sorry @andrewmlim I accidentally got a space into the commit message. Can you please close this branch?

**Comment by andrewmlim** [comment]
Closing.

**Pull request closed without merging**